test/unit/specs/components/SelectedLicense.spec.js

```diff
@@ -0,0 +1,28 @@
+import { shallowMount } from '@vue/test-utils'
+import SelectedLicense from '@/components/SelectedLicense.vue'
+
+describe('SelectedLicense.spec.js', () => {
+    let cmp
+
+    beforeEach(() => {
+        cmp = shallowMount(SelectedLicense, {
+        // Be aware that props is overridden using `propsData`
+            propsData: {
+                fullLicenseName: 'Attribution',
+                shortLicenseName: 'BY'
```

Comment:
Please format shortLicenseName as "CC {name} 4.0". For example, BY would be "CC BY 4.0", BY-ND would be "CC BY-ND 4.0". This is how the prop is actually formatted when passed to the component.